client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+const Suggestions = require( './suggestions-mixin' );
```

Comment:
If this is a brand new mixin, would it be possible to rewrite this as a higher order component? Feel free to ping me if you have any questions

https://facebook.github.io/react/blog/2016/07/13/mixins-considered-harmful.html#higher-order-components-explained